app/controllers/thesis_controller.rb

```diff
@@ -43,7 +43,7 @@ def publication_statuses
     @terms = defined_terms Thesis.all
     @publication_statuses = Thesis.all.pluck(:publication_status).uniq.sort
     # Filter relevant theses by selected term from querystring
-    term_filtered = filter_theses_by_term Thesis.all
+    term_filtered = filter_theses_by_term Thesis.all.includes(:degrees).includes(:departments).includes(:users)
```

Comment:
Is there a difference between this and `.includes(:degrees, departments, users)`?

Reply:
No idea! I can try it that way if it reads cleaner to you